src/components/Menus/MainMenu/useMainMenuContent.tsx

```diff
@@ -422,6 +412,22 @@ export const useMainMenuContent = () => {
         });
       },
     },
+    {
```

Comment:
Can be removed

Reply:
Remove all commented code, left the comment about rendering menu on the server: 

> //Todo: to generate on the server side